python/cudf/cudf/core/groupby/groupby.py

```diff
@@ -2298,9 +2304,15 @@ def pct_change(
             Periods to shift for forming percent change.
         fill_method : str, default 'ffill'
             How to handle NAs before computing percent changes.
+
+            .. deprecated:: 23.12
```

Comment:
So recently, supported was added back for `fill_method=None` which means "don't fill the NA values" https://github.com/pandas-dev/pandas/pull/55527/